sdk/core/Azure.Core/src/RehydrationToken.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable enable
```

Comment:
Nullable annotations are turned on for Azure.Core, so we shouldn't need this statement.

Reply:
Right, will remove it.